src/types.json

```
@@ -0,0 +1,79 @@
+{
+  "Terms of service": {
```

Let's use title case everywhere 🙂

```suggestion
  "Terms of Service": {
```